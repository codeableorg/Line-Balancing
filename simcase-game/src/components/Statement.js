/** @jsx jsx */
import { jsx } from "@emotion/core";

const center = {
  textAlign: "center",
  backgroundColor: "rgb(213,245,245)",
  marginLeft: "auto",
  marginRight: "auto",
  width: "90%"
};
const textStyles = {
  color: "rgb(110,140,120)",
  fontFamily: "Arial",
  fontSize: "1.5em"
};

function Statement({
  statementText = "Balance production to increase production. Use the radio buttons to shift tasks between adjacent workers. Good Luck!"
}) {
  return (
    <div css={center}>
      <h1 css={textStyles}>{statementText}</h1>
    </div>
  );
}

export default Statement;
