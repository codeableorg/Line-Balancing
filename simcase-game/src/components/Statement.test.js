import React from "react";
import { render } from "@testing-library/react";

import Statement from "./Statement";
test("Statement based on game stage", () => {
  const { asFragment } = render(<Statement />);
  expect(asFragment()).toMatchSnapshot();
});
