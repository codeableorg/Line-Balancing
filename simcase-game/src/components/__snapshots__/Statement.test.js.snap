// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Statement based on game stage 1`] = `
<DocumentFragment>
  .emotion-1 {
  text-align: center;
  background-color: rgb(213,245,245);
  margin-left: auto;
  margin-right: auto;
  width: 90%;
  border-radius: 4px;
  padding: 1px 1em;
  box-sizing: border-box;
  margin-bottom: 16px;
  font-size: 1.2em;
}

.emotion-0 {
  color: green;
  font-family: Arial;
}

<div
    class="emotion-1"
  >
    <p
      class="emotion-0"
    >
      Balance production to increase production. Use the radio buttons to shift tasks between adjacent workers. Good Luck!
    </p>
  </div>
</DocumentFragment>
`;
